content/akeneo-web-api.yaml

```diff
@@ -2157,6 +2291,61 @@ definitions:
     allOf:
       - $ref:  '#/definitions/ItemList'
       - $ref: '#/definitions/AttributeOption'
+  AttributeGroup:
+    type: object
+    properties:
+      code:
+        type: string
+        description: Code of attribute group
+      sort_order:
+        type: integer
+        description: Order of the attribute group
+        x-validation-rules: It is a positive integer
+        default: 0
+      attributes:
+        type: array
+        description: Attributes codes that compose the attribute group
+        x-validation-rules: Each string of the array is an existing attribute code
```

Comment:
Missing rule "An attribute can only be in one attribute group." Modulo the English.